src/app/Dashboard/LayoutTemplateUploadModal.tsx

```diff
@@ -162,7 +162,7 @@ export const LayoutTemplateUploadModal: React.FC<LayoutTemplateUploadModalProps>
                   onSingleSuccess(
                     fileUpload.file.name,
                     <Text component="p" style={{ color: 'var(--pf-global--warning-color--200)' }}>
-                      Warning: To see this template in the template picker, make sure the Cryostat Feature Level is set
+                      Warning: To see this template in the template picker, make sure the Cryostat Feature level is set
```

Comment:
I think `feature` should be lowercased too here

Reply:
Also here to match the UI 
![image](https://github.com/cryostatio/cryostat-web/assets/56743004/1a630a8e-ca74-4110-a681-db3314c408fd)